charts/jupyterhub/LICENSE

```diff
@@ -0,0 +1,238 @@
+BSD 3-Clause License
```

Comment:
Is this combination of BSD and Apache from upstream?

Reply:
Yep!  Exact upstream license.

Reply:
I was about 95% sure, but it's such a weird choice by them. Not sure that even works legally. Oh well.